plugin/main/src/kotlinx/benchmark/gradle/BenchmarksExtension.kt

```diff
@@ -97,4 +97,4 @@ open class BenchmarksExtension(val project: Project) {
             }
         }
     }
-}
+}
```

Comment:
Please try to avoid changes like this. It adds a needless entry to the file history.

Reply:
Noted. I am curious as to what the root cause of this showing up in the file history is given their is no change. Will be wary moving forward.